std_msgs/String pose_topic
---
std_msgs/Bool success
